frontend/cypress/integration/containers.spec.js

```diff
@@ -49,6 +49,41 @@ export const containersTests = () => {
         cy.get('button').contains('Go Back').click()
         cy.get('body').should('contain', '1-10 of 15 items')
       })
+      
+      const moveBarcodeDst = 'freezer-three';
+      it('moves containers (template import)', () => {
+        cy.navigateTo('Container', 'Move Containers')
+        cy.get('input[type=file]').attachFile('Container_move_v3_2_0_F_A_1.xlsx')
+        cy.submitForm()
+        cy.get('.ant-alert-success').should('contain', 'Template submitted')
+        cy.get('button').contains('Go Back').click()
+        cy.get('body').should('contain', '1-10 of 15 items')
+        cy.get('.ant-table-thead .ant-table-filter-trigger').eq(1).click().type(moveBarcodeDst) // filter by barcode (column 1)
+        cy.get('.ant-table-tbody .ant-table-row').within(() => cy.get('.ant-table-cell').eq(5).should('contain', '2'))
+        cy.get('button').contains('Clear Filters').click()
+      })
+
+      const renameBarcodeSrc = 'freezer-three';
+      const renameBarcodeDst = 'freezer-four';
+      const renameNameDst = 'freezer-4';
+      it('renames containers (template import)', () => {
+        cy.navigateTo('Container', 'Rename Containers')
+        cy.get('input[type=file]').attachFile('Container_rename_v3_2_0_F_A_1.xlsx')
+        cy.submitForm()
+        cy.get('.ant-alert-success').should('contain', 'Template submitted')
+        cy.get('button').contains('Go Back').click()
+        cy.get('body').should('contain', '1-10 of 15 items')
+        cy.get('.ant-table-thead .ant-table-filter-trigger').eq(1).click().type(renameBarcodeSrc)// filter by barcode (column 1)
+        cy.get('body').should('contain', '0-0 of 0 items')
+        cy.get('button').contains('Clear Filters').click()
+        cy.get('.ant-table-thead .ant-table-filter-trigger').eq(1).click().type(renameBarcodeDst) // filter by barcode (column 1)
+        cy.get('body').should('contain', '1-1 of 1 items')
+        cy.get('button').contains('Clear Filters').click()
+        cy.get('.ant-table-thead .ant-table-filter-trigger').eq(0).click().type(renameNameDst) // filter by name (column 0)
+        cy.get('body').should('contain', '1-1 of 1 items')
+        cy.get('button').contains('Clear Filters').click()
+      })
+
```

Comment:
Unless the variables are shared, they should be in the smallest scope required.

Reply:
ok... i moved them inside the inner scope.